db/schema.rb

```diff
@@ -11,18 +11,18 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20141229224941) do
+ActiveRecord::Schema.define(version: 20150306182232) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "active_admin_comments", force: :cascade do |t|
-    t.string   "namespace",     limit: 255
```

Comment:
Yes I would also be skeptical of that advice, unless there's something big I'm totally missing this whole time.  Also I've noticed a gem (`https://github.com/ActsAsParanoid/acts_as_paranoid`) can break migrations as well because it expects columns (`deleted_at`) to be present even if the migration hasn't happened yet.  It hooks this code so deep into AR that it's even called from the model while migrations run.